book/src/quickstart/rust.md

```diff
@@ -12,25 +12,28 @@
 
 Setting up just requires you to [install cargo-dist][install] and then run `cargo dist init` in your [Cargo workspace][workspace]. This command interactively walks you through configuration options, **and should be run again whenever you want to change your settings or want to update cargo-dist**.
 
-Just to really emphasize that: `cargo dist init` is designed to be rerun over and over, and will preserve your settings while handling any necessary updates and migrations. Always Be Initing.
+Just to really emphasize that: [`cargo dist init` is designed to be rerun over and over, and will preserve your settings while handling any necessary updates and migrations. Always Be Initing](../updating.md).
 
 
 
 ### Initial Setup
 
-Since this is the *way-too*-quickstart, we pass `--yes` to auto-accept all defaults on our first setup!
+Since this is a quickstart, we'll pass `--yes` to auto-accept all defaults on our first setup!
 
 ```sh
-# install tools (build from source is the most portable option)
-cargo install cargo-dist
-
 # setup cargo-dist in your project (--yes to accept defaults)
 cargo dist init --yes
 git add .
 git commit -am "chore: wow shiny new cargo-dist CI!"
 ```
 
-The one-time setup will add a decent default configuration to your root Cargo.toml and generate CI for orchestrating itself in `.github/workflows/release.yml`. If the CI file isn't created, this probably means you don't have `repository = "https://github.com/..."` consistently set in your Cargo.toml(s).
+**It's very common for `cargo dist init` to return an error about the "repository" URLs set in your Cargo.toml. If this happens, no work will be lost.** You can just follow the instructions in the error and rerun `cargo dist init` again and it will pick up where you left off.**
```

Comment:
There's a stray `**` at the end here: 
```suggestion
**It's very common for `cargo dist init` to return an error about the "repository" URLs set in your Cargo.toml. If this happens, no work will be lost.** You can just follow the instructions in the error and rerun `cargo dist init` again and it will pick up where you left off.
```